watertap/unit_models/tests/test_translator_adm1_asm1.py

```diff
@@ -0,0 +1,303 @@
+            94.468 * units.mmol / units.liter * 14 * units.mg / units.mmol
+        )
+        m.fs.unit.inlet.conc_mass_comp[0, "S_I"].fix(130.87 * units.mg / units.liter)
+
+        m.fs.unit.inlet.conc_mass_comp[0, "X_c"].fix(107.92 * units.mg / units.liter)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_ch"].fix(20.517 * units.mg / units.liter)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_pr"].fix(84.22 * units.mg / units.liter)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_li"].fix(43.629 * units.mg / units.liter)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_su"].fix(312.22 * units.mg / units.liter)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_aa"].fix(931.67 * units.mg / units.liter)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_fa"].fix(338.39 * units.mg / units.liter)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_c4"].fix(335.77 * units.mg / units.liter)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_pro"].fix(101.12 * units.mg / units.liter)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_ac"].fix(677.24 * units.mg / units.liter)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_h2"].fix(284.84 * units.mg / units.liter)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_I"].fix(17216 * units.mg / units.liter)
+
+        m.fs.unit.inlet.cations[0].fix(1e-8 * units.mmol / units.liter)
+        m.fs.unit.inlet.anions[0].fix(5.21 * units.mmol / units.liter)
+
+        return m
+
+    @pytest.mark.build
+    @pytest.mark.unit
+    def test_build(self, asmadm):
+
+        assert hasattr(asmadm.fs.unit, "inlet")
+        assert len(asmadm.fs.unit.inlet.vars) == 6
+        assert hasattr(asmadm.fs.unit.inlet, "flow_vol")
+        assert hasattr(asmadm.fs.unit.inlet, "conc_mass_comp")
+        assert hasattr(asmadm.fs.unit.inlet, "temperature")
+        assert hasattr(asmadm.fs.unit.inlet, "pressure")
+        assert hasattr(asmadm.fs.unit.inlet, "anions")
+        assert hasattr(asmadm.fs.unit.inlet, "cations")
+
+        assert hasattr(asmadm.fs.unit, "outlet")
+        assert len(asmadm.fs.unit.outlet.vars) == 5
+        assert hasattr(asmadm.fs.unit.outlet, "flow_vol")
+        assert hasattr(asmadm.fs.unit.outlet, "conc_mass_comp")
+        assert hasattr(asmadm.fs.unit.outlet, "temperature")
+        assert hasattr(asmadm.fs.unit.outlet, "pressure")
+        assert hasattr(asmadm.fs.unit.outlet, "alkalinity")
+
+        assert number_variables(asmadm) == 129
+        assert number_total_constraints(asmadm) == 16
+
+        assert number_unused_variables(asmadm.fs.unit) == 4
+
+    @pytest.mark.component
+    def test_units(self, asmadm):
+        assert_units_consistent(asmadm)
+
+    @pytest.mark.unit
+    def test_dof(self, asmadm):
+        assert degrees_of_freedom(asmadm) == 0
+
+    @pytest.mark.solver
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_initialize(self, asmadm):
+        initialization_tester(asmadm, optarg={"bound_push": 1e-8})
```

Comment:
Is the bound_push necessary?

Reply:
```suggestion
        initialization_tester(asmadm)
```
I checked, and it isn't.